transcripts/AttachmentPromptTests/BasicAttachmentPrompt.transcript

```diff
@@ -0,0 +1,87 @@
+[
```

Comment:
This should be auto-generated at test time right? 

the .chat file is the source of truth, and that can be (on the fly) converted to a transcript file and executed. I don't think it makes sense to have both sets of files checked in. That adds complexity and doesn't seem to bring any benefits. 

Let's discuss, as I may be missing something.

Reply:
To the first comment: In this case (Dialogs.Tests) yes, the rule was to create the .chat and simply use Chatdown to generate the .transcript file. Except for one case. In other cases like Prompts.Tests, there are several cases where the .transcript file was edited by hand.
Initially, we thought about including both .chat and .transcript files as a guideline of how we created them.

Reply:
For this specific file the .chat-to-.transcript conversion was not enough. We had to manually edit the resulting .transcript file to include the `attachments` property with the corresponding content values.